src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/exceptions/RefundNotAllowedException.kt

```diff
@@ -0,0 +1,8 @@
+package it.pagopa.ecommerce.eventdispatcher.exceptions
+
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.ResponseStatus
+
+@ResponseStatus(value = HttpStatus.BAD_GATEWAY)
+class RefundNotAllowedException(detail: String) :
```

Comment:
Why do you pass a detail if it is not used ?

Reply:
Right, solved with [e319b38](https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/109/commits/e319b3809f5fc94abcbd7503b350a7bf95e4eaf7) :)